### DISCUSSION TITLE
Metas e performance individual e de equipe.

### DISCUSSION THREAD

**Discussion by WarleyFranco** [frontendbr/forum#995] [Discussão]
Oi pessoal, quero saber um pouco como a empresa de vcs trabalha com essa questão de **meta e performance** (tanto individual quanto de equipe), qual sistema, metodologia/método para metrificar.

Também seria bacana a opinião de vcs a respeito do que vcs entendem por meta e performance, os prós e contras de metas, o que vocês acham em geral.

**Comment by WarleyFranco** [comment]
Atualmente onde trabalho, não utilizamos nenhuma metodologia ou métrica bem definida para as metas. 

Estamos pensando em iniciar algo mais bem desenhado, e quero entender o que é legal e o que a comunidade tem usado pra levar como sugestão.

Na minha cabeça é válido nós aderirmos algo que é sólido e bastante usado tanto pela comunidade quanto pelo mercado.

**Comment by julioxavierr** [comment]
Eu uso para definir metas pessoais as metas SMART, é bem interessante, dá até um estímulo a mais. Acredito que seja bastante usado, como você busca.

**Comment by ghost** [comment]
Já trabalhei em lugares que usavam KPI e OKR. O que mais achei bacana foi OKR, pois em sistema de squad vc pode ate colocar metas de "guilds". Adotei na vida a metodologia de OKR achei fantástica.

**Comment by WarleyFranco** [comment]
@julioxavierr, gostei do conceito da SMART, me parece orientar bem o início da coisa, no ambiente de trabalho vc utiliza também? Tem uma boa aderência/engajamento?

@edmolima, não conhecia o OKR, já estou dando uma pesquisada e achei bem completona, vou mais a fundo e dou um reply aqui.

**Comment by Cleysonlb** [comment]
Aqui onde trabalho começamos a usar o OKR a uns 2 meses, ainda não fechamos um trimestre mas aparentemente esta surgindo resultados, a meta do OKR aqui é por setor e não colocamos meta individual pois acreditamos que as metas tem que ser alcançadas como time!

**Comment by WarleyFranco** [comment]
@Cleysonlb, eu acredito bastante nessa questão da meta por equipe, bacana vcs estarem utilizando OKR, eu tô dando uma olhada e tô curtindo também, levaram isso para o escopo geral da empresa ou apenas para dentro de um time?

@edmolima, posso estender a pergunta pra vc tbm? Rs

**Comment by Cleysonlb** [comment]
Cara aqui usamos na empresa como um todo, assim fazemos uma reunião quinzenal para analisar os dados, mas pode ser implementado somente no time de desenvolvimento, geralmente os KPIS do OKR estão alinhados com as metas da empresa, então tem que ver direitinho.

**Comment by fdaciuk** [comment]
Na última empresa que trabalhei, usávamos OKR mesmo, para a empresa toda, não somente para o time de desenvolvimento.

Funcionava muito bem =)

**Comment by caioincau** [comment]
Usamos KPIs e OKRs para time/empresa e um sistema de avaliação de desempenho, colocando metas individuais e de alinhamento com a cultura de empresa, inclusive isso é algo que influencia mto nas promoções por aqui.

**Comment by WarleyFranco** [comment]
@Cleysonlb muito bom brother, obrigado pela explanação.

@fdaciuk, maravilhoso acho que pode ser algo bem bacana pq já temos metas na empresa como um todo.

**Comment by WarleyFranco** [comment]
@caioincau muito legal, transformar isso pra algo macro e aplicar essas métricas para promoções. A questão do desempenho, utilizam algo além do OKR pra metrificar?

**Comment by caioincau** [comment]
Não, mais OKR e alinhamento com a cultura/job description, tem uma escala de 0-5, de quanto você atende do seu Job Description por exemplo, sendo 0 péssimo, 3 atende plenamente e 5 faz mais do que o esperado, inspirando pessoas ao seu redor.

A régua é bem alta

**Comment by WarleyFranco** [comment]
@caioincau lindo isso! Acho que encaixa perfeitamente com o que a empresa tem como valores, vou dar uma levantada e levar pra diretoria um pouco dessa metodologia e viabilizar uma POC do OKR pra nossa realidade.

**Comment by ghost** [comment]
@WarleyFranco como muitos falaram o comum é realmente ter OKR de times e empresa como um todo. Essas metas cada empresa dá seu peso por exemplo para uma bonificação.
